src/cfnlint/customRules/Rule.py

```diff
@@ -22,8 +22,8 @@ class Rule(object):
     error_message = ''
 
     def __init__(self, line, ruleNumber, ruleSet):
-        line = line.replace('"', '')
-        self.lineNumber = ruleNumber
+        line = line.rstrip().replace('"', '')
+        self.lineNumber = ruleNumber + 9000
```

Comment:
?

Reply:
rstrip removes the newline character - it was causing the space between the error message and the line numbers.

Basically resolves the conflict between two seperate mechanisms for detecting the start of a custom error message

Reply:
Oh i meant the + 9000

Reply:
Oooh - [(E|W|I)9xxx is 'reserved for users rules'](https://github.com/aws-cloudformation/cfn-python-lint/blob/master/docs/rules.md#categories)

That's the ruleID assignment - Where it was agreed to use the 9000 range for custom rules. As ruleID for custom rules are based on line number, this just adds 9000 to that value.